src/main/java/org/kcctl/command/InfoCommand.java

```diff
@@ -23,12 +23,21 @@
 import org.kcctl.util.ConfigurationContext;
 
 import picocli.CommandLine.Command;
+import picocli.CommandLine.Model.CommandSpec;
+import picocli.CommandLine.Spec;
 
 @Command(name = "info", description = "Displays information about the Kafka Connect cluster")
 public class InfoCommand implements Runnable {
 
+    private final ConfigurationContext context;
+
+    @Spec
+    CommandSpec spec;
```

Comment:
So can we get this via the constructor too? Would be nice if we could make all the fields `private final`.

Reply:
Not easily, since then we would have to register picocli CommandSpecs for quarkus to somehow be injectable. I could take a deeper look into that.

Reply:
I've looked into this a little bit. However, I couldn't find an easy/direct way to make this work with Quarkus. It seems that supporting this in PicoCLI directly is the best option.